src/renderer/home/styles.scss

```diff
@@ -110,8 +110,20 @@ html, body, #app {
     text-transform: uppercase;
     letter-spacing: -.05em;
     line-height: 1;
+    top: 10px;
   }
 
+  .headerAndSettings {
+     display: flex;
+     align-items: baseline;
+     flex-direction: row;
+     justify-content: flex-end;
+   }
+
+   .settingsButton {
+     margin-left: 0.3 rem;
```

Comment:
Convention: no space between the value and the unit in numeric CSS property values: `margin-left: 0.3rem;`